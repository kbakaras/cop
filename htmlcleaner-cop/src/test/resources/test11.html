<html>
<head>
 <script>function doStuff(){
  alert("<hello!>");
 }</script>
 <style><![CDATA[<><><>]]></style>
</head>
<body>
<p><![CDATA[test]]></body>
</html>