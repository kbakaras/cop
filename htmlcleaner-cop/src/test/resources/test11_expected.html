<html>
<head>
 <script>/*<![CDATA[*/
function doStuff(){
  alert("<hello!>");
 }
/*]]>*/</script>
 <style>/*<![CDATA[*/
<><><>
/*]]>*/</style>

</head>
<body>

<p>&lt;![CDATA[test]]&gt;
</p></body></html>